Makefile

```diff
@@ -154,7 +154,7 @@ e2e-build-run-k8s: build test-e2e-k8s
 ################################################################################
 .PHONY: test-e2e-upgrade
 test-e2e-upgrade: test-deps
-	gotestsum --jsonfile $(TEST_OUTPUT_FILE) --format standard-verbose -- -timeout 20m -count=1 -tags=e2e ./tests/e2e/kubernetes/... -run="TestUpgradePath"
+	gotestsum --jsonfile $(TEST_OUTPUT_FILE) --format standard-verbose -- -timeout 30m -count=1 -tags=e2e ./tests/e2e/upgrade/...
```

Comment:
For now setting it to 30m but it takes around 25+ minutes to complete. 
Have opened a separate issue to track reduction of time for the upgrade path tests: https://github.com/dapr/cli/issues/724